.github/workflows/ci.yml

```diff
@@ -0,0 +1,40 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  test:
+    name: Test
+
+    strategy:
+      matrix:
+        ocaml-compiler:
+          - 4.12.x
+          - 4.13.x
```

Comment:
I think we can skip `4.13`. I'd suggest oldest supported version, newest 4.x version, newest 5.x version for the selections.